routers/web/user/package.go

```diff
@@ -376,6 +377,10 @@ func PackageSettingsPost(ctx *context.Context) {
 			ctx.Flash.Success(ctx.Tr("packages.settings.delete.success"))
 		}
 
+		if err := cleanup_service.PostPackageRemoval(ctx, ctx.Package.Descriptor); err != nil {
```

Comment:
Why it's not a part of `RemovePackageVersion`?

Reply:
done